src/Widgets/ThumbnailGalleryWidget/thumbnailGalleryWidget.scss

```diff
@@ -0,0 +1,120 @@
+@import "src/assets/stylesheets/variables";
+@import "src/assets/stylesheets/mixins";
+@import "src/assets/stylesheets/bootstrap/bootstrap";
```

Comment:
Adding the complete `bootstrap` to a widget's css is not a good idea. This instructs sass and webpack to add a full copy of bootstrap to the final `index.css`. With this is actually got really big (see https://travis-ci.org/Scrivito/scrivito_example_app_js/jobs/569311355#L304) compared to the current master (see https://travis-ci.org/Scrivito/scrivito_example_app_js/jobs/569301877#L294). So increasing the `index.css` by over 50% (more than 100 kb) is not an option.

If you do make use of bootstrap, please only import it partially and ideally only mixins (e.g. `@import "src/assets/stylesheets/bootstrap/mixins/clearfix";`).

Also: Whenever you're using a `@extend` directive in a widget's scss file please try to replace it with a `@include` rule or a plain css style. Otherwise we run into the risk of overwriting the original `@extend` method, as previously experienced with the `@extend fa` in a previous PR.